src/JustSaying/AwsTools/MessageHandling/Dispatch/MessageDispatcher.cs

```diff
@@ -197,12 +233,15 @@ private async Task UpdateMessageVisibilityTimeout(IQueueMessageContext messageCo
             }
         }
 
-        private static bool TryGetApproxReceiveCount(IDictionary<string, string> attributes, out int approxReceiveCount)
+        private static bool TryGetApproxReceiveCount(
+            IDictionary<string, string> attributes,
+            out int approxReceiveCount)
         {
             approxReceiveCount = 0;
 
-            return attributes.TryGetValue(MessageSystemAttributeName.ApproximateReceiveCount, out string rawApproxReceiveCount) &&
-                   int.TryParse(rawApproxReceiveCount, out approxReceiveCount);
+            return attributes.TryGetValue(MessageSystemAttributeName.ApproximateReceiveCount,
+                    out string rawApproxReceiveCount) &&
+                int.TryParse(rawApproxReceiveCount, out approxReceiveCount);
```

Comment:
Should use the overload that accepts `CultureInfo.InvariantCulture`

Reply:
Done, the overload also specifies int number style.